targets/esp8266/jshardware.c

```diff
@@ -588,15 +593,47 @@ void jshUSARTKick(
 //===== SPI =====
 
 /**
- * Unknown
+ * Initialize the hardware SPI device.
+ * On the ESP8266, hardware SPI is implemented via a set of pins defined
+ * as follows:
  *
+ * | GPIO   | NodeMCU | Name  | Function |
+ * |--------|---------|-------|----------|
+ * | GPIO12 | D6      | HMISO | MISO     |
+ * | GPIO13 | D7      | HMOSI | MOSI     |
+ * | GPIO14 | D5      | HSCLK | CLK      |
+ * | GPIO15 | D8      | HCS   | CS       |
```

Comment:
That's a good question.  This table and documentation comes from ESP8266 "stuff".   I think that when one "drives" the ESP8266 hardware SPI, the SPI hardware knows to drive CS.  I suspect that if we wanted to drive alternate devices, we would ignore the hardware CS and dedicate alternate pins to CS.